### ISSUE TITLE
[Minor typo I think] https versus http links on the homepage at: https://pngquant.org/releases.html

### ISSUE THREAD

**Issue by rubyFeedback** [kornelski/pngquant#382]
Heya kornelski,

If you go to:

https://pngquant.org/releases.html

You see a nice overview of available releases.

If, however had, you try to download one, you get a non-https link such as:

http://pngquant.org/pngquant-2.15.1-src.tar.gz

Should this not also be https there? I am asking in particular because if
you go to http://pngquant.org/releases.html you are automatically
re-directed onto https://pngquant.org/releases.html. So I would naively
assume that the downloads are the same.

(Background how I found this: I actually register all downloadable URLs
in a ruby project in general; I saw on another website that there is a new
release of pngquant, so I checked it. I try to prefer https these days, 
but my old entries still had http, so I wanted to see whether https is
available. Note that https://pngquant.org/pngquant-2.15.1-src.tar.gz
also works, so my naive assumption here is that the .html page listed
above has not yet made the switch, unless of course you decided to
do so deliberately, in order to allow for older browsers to download
or something. Still, then, it is weird that http://pngquant.org/releases.html
is an automatic redirect towards https://pngquant.org/releases.html so
I really have no idea. I am just reporting this anyway, and yes it is
so minor that I wonder whether I should even report it. Anyway
cheers! Long live pngquant REDUCE THE FILE SIZE OF EVERYTHING!!!)

**Comment by kornelski** [comment]
Yes, it was meant to be HTTPS link. The site has HSTS set, so your browser should be using HTTPS for it anyway.
